components/NavigationDrawer/src/private/MDCBottomDrawerContainerViewController.m

```diff
@@ -284,7 +304,7 @@ - (BOOL)isMobileLandscape {
 }
 
 - (BOOL)shouldPresentFullScreen {
-  return [self isAccessibilityMode] || [self isMobileLandscape];
+  return [self isAccessibilityMode] || [self isMobileLandscape] || _initialDrawerFactor >= 1;
```

Comment:
were you able to figure out the correct number here rather than 1? if not, please open a tracking bug for follow up work

Reply:
Bug was added.